lib/backup.rb

```diff
@@ -25,6 +25,11 @@ def backup(board_id)
   def list
     Dir.entries(@directory).reject { |d| d =~ /^\./ }
   end
+  
+  def show_json(board_id)
```

Comment:
what about renaming `show_json` to `show_diff(local_board_id, online_board_id)` and move here the [put](https://github.com/openSUSE/trollolo/pull/196/files#diff-8210eb1811886a2c9582cb7d37944ee0R245) you have in cli? I think that code is really related to the backup and should be in the `Backup` model :smile:

Reply:
I also wonder if you should do something like what we have in the [`Backup#show` command](https://github.com/openSUSE/trollolo/pull/196/files#diff-9e2d49cc12b971f42cce40f4918d4ca7R35): 

```ruby
out = options[:output] || STDOUT
```